src/explore-education-statistics-admin/src/pages/release/datablocks/components/chart/ChartReferenceLinesConfiguration.tsx

```diff
@@ -187,16 +236,59 @@ export default function ChartReferenceLinesConfiguration({
               <tr>
                 <td className="dfe-vertical-align--bottom">
                   {type === 'major' ? (
-                    <FormFieldSelect
-                      name="position"
-                      id="referenceLines-position"
-                      label="Position"
-                      formGroup={false}
-                      hideLabel
-                      placeholder="Choose position"
-                      order={FormSelect.unordered}
-                      options={filteredOptions}
-                    />
+                    <>
+                      <FormFieldSelect
+                        name="position"
+                        id="referenceLines-position"
+                        label="Position"
```

Comment:
As mentioned in other discussions, I'm not keen on implementing this so that the reference line renders on the opposite axis to where it's actually been added. It's counter-intuitive and is really obvious if the lines are styled (they clearly run at a tangent to the axis the user is working on), and also means that we have add flags all over the code to toggle behaviour (a code smell).

Would suggest that we rejig things so that these bounded reference lines are implemented on the correct axis.

For example, on a line chart, we'd have something like this on the Y axis (minor) tab:

![image](https://github.com/dfe-analytical-services/explore-education-statistics/assets/9917868/75aab1cb-4378-43be-82c5-c016e5e0d290)

The 'Custom' option would work how it does currently, being a percentage along the other axis.

I think ideally we'd also allow none percentage positions to be set as well to really dial in the UX, but we can leave that for another day.

This change would need some modifications to the data structure i.e. `otherAxisStart` and `otherAxisEnd` instead of `startPosition` and `endPosition`.

Reply:
It's moved to the minor axis now.